zabbix_auto_config/processing.py

```diff
@@ -642,6 +651,18 @@ def do_update(self):
                 if zabbix_host["inventory"]["inventory_mode"] != "1":
                     self.set_inventory(zabbix_host, 1)
 
+            if "tags" in db_host and len(db_host["tags"]) > 0:
+                zabbix_host_tags = set(tuple(sorted(d.items())) for d in zabbix_host["tags"])
+                db_host_tags = set(tuple(sorted(d.items())) for d in db_host["tags"])
+
+                if not db_host_tags.issubset(zabbix_host_tags):
```

Comment:
I don't think this block does the right thing. It's also a complicated? Do a merge in `tag_host` instead?

Also: Tags in DB should be enforced(?). As stated: I don't think this part works.

Reply:
This code is tested in a test env. and works. Reason for doing a merge here not in the `tag_host` is to be consistent as preparation & modifications for "proxy_pattern", "interfaces" and "inventory_mode" is done above this code and relevant functions are called afterwards.
Enabling users to tag hosts which are to be tagged by zac at the same time makes it complicated unfortunately. Furthermore I found out that I need to improve this part of code in order to better distinguish tags added by users and tags added by zac. For this purpose, we are going to "_" in tags names added by zac. 
Improved version of code will be available soon for review.